parts/linux/cloud-init/artifacts/containerd-monitor.timer

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=a timer that delays containerd-monitor from starting too soon after boot
+[Timer]
```

Comment:
do we not need to specify the service name here?

Reply:
ok, it's using the naming convention, but better to be explicit IMO

Reply:
yea i'll add the unit name to be explict